test.js

```diff
@@ -0,0 +1,15 @@
+var request = require('request');
+
+request.post({
+    url: 'http://localhost:8000/metadata',
+    method: 'POST',
+    json: { test: "test" }
```

Comment:
This does work, maybe it's a version thing? I feel like I tried your version and was having issues.